src/actions/validate.js

```diff
@@ -26,28 +27,32 @@ const validateData = (schema, data) => {
   }
 };
 
-const generateSchema = (requiredLanguages, optionalLanguages) => {
+const generateSchema = (requiredLanguages, optionalLanguages, collections) => {
   let schema = schemaTemplate;
+  
+  const upperCaseCollections = collections.map(s => s.toUpperCase());
 
+  // To check if all requested collections are present in the YAML file.
```

Comment:
Here we're modifying the validation schema to make sure an error is thrown if one of the requested collections does not exist in the Localicipe.